### ISSUE TITLE
Support for non-English texts

### ISSUE THREAD

**Issue by jcuenod** [JonathanReeve/text-matcher#12]
I guess that the key code to check would be the tokenizer (which doesn't look like it would give problems) but do you have any idea about whether this would work on non-English texts (I'm thinking specifically of Hebrew but I would be interested in Greek as well)?

**Comment by JonathanReeve** [comment]
Good idea. Is this for the CLI app, by the way, or the app as used in a Python module? What I could do is add an option to the CLI to change the tokenizer to a simpler whitespace-based one. It wouldn't work well with languages that lack whitespace, like Chinese, and certain varieties of ancient Greek and Hebrew, I'm sure, but it would allow for using this with a lot of other European languages. (And if you had a way to segment those languages that don't have whitespace into text files that contain whitespace, it would probably work.)

**Comment by jcuenod** [comment]
A whitespace tokenizer would be perfect, yes. That way I could generate a text full of lexemes rather than a fully inflected text and feed that in as the text.

**Comment by JonathanReeve** [comment]
So I took a look and it seems I already tokenize using a [regexp](https://github.com/JonathanReeve/text-matcher/blob/b6931761e94f6925cfaea9e1f4e27b8f4f6c334d/text_matcher/matcher.py#L36). Could you tell me more about what is/isn't working for you? I imagine maybe the stemmer/lemmatizer might break words strangely in other languages? I can add an option to turn this off, at least.
